src/intTest/java/ca/nrc/cadc/nameresolver/integration/VosiCapabilitiesTest.java

```diff
@@ -100,7 +100,7 @@
  */
 public class VosiCapabilitiesTest extends CapabilitiesTest {
     private static final Logger log = Logger.getLogger(VosiCapabilitiesTest.class);
-    private static final URI RESOURCE_ID = URI.create("ivo://cadc.nrc.ca/resolver");
+    private static final URI RESOURCE_ID = URI.create("ivo://opencadc.org/resolver");
```

Comment:
The registry still has this as `ivo://cadc.nrc.ca/resolver`, so integration tests won't run out of the box anymore.

Reply:
In general this is supposed to run against local instance. We use `opencadc.org` to distinguish from the release `cadc.nrc.ca` and prevent destructive int tests from accidentally wreaking havoc production. This is a special case where int tests can safely be run against production. Should we make an exception from the general rule or is there a better approach?

Reply:
Then we should make this configurable with an environment variable or something.  There's no way to run the tests without making a code change with this change, or forcing the user to run their own registry.

Reply:
Agree and I've tried. Unfortunately, `VosiCapabilitiesTest` is very dependent on `reg`. Either disable that or leave it as it is and deal with it when an external user wants to use it.